src/components/Reports/GoalCalculator/SharedComponents/SectionList.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import CircleIcon from '@mui/icons-material/Circle';
+import RadioButtonUncheckedIcon from '@mui/icons-material/RadioButtonUnchecked';
+import {
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  SxProps,
+  Theme,
+  styled,
+} from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { GoalCalculatorReportEnum } from '../GoalCalculatorHelper';
+import { useGoalCalculator } from '../Shared/GoalCalculatorContext';
+
+const categoryListItemStyles: SxProps<Theme> = (theme) => ({
+  '.MuiSvgIcon-root': {
+    fontSize: '1rem',
+  },
+  padding: 0,
+  paddingLeft: theme.spacing(3),
+});
+
+const CategoryListItemIcon = styled(ListItemIcon)(({ theme }) => ({
+  minWidth: 'auto',
+  marginRight: theme.spacing(0.5),
```

Comment:
[Mentoring] Good use of styled components for consistent spacing. The `paddingLeft: theme.spacing(3)` and `marginRight: theme.spacing(0.5)` patterns follow Material-UI spacing conventions. Consider extracting common list item styling patterns if this grows to support more complex list variations.

Reply:
I've tried removing these spacing styles, but they are needed to achieve the layout that we want.